crates/rolldown/tests/esbuild/importstar/export_other_common_js/artifacts.snap

```diff
@@ -18,11 +18,10 @@ var require_foo = __commonJSMin((exports) => {
 var import_foo = __toESM(require_foo());
 
 //#endregion
-var bar = import_foo.bar;
```

Comment:
This var only used when reexport, inline it will generate less  code, and also align with esbuild.

Reply:
here is a example, if we use `bar` and then reexoprt it, 

```js
import {bar} from './foo'

console.log(`bar: `, bar)
export {bar}
```
here is the result: 

```js

//#region foo.js
var require_foo = __commonJSMin((exports) => {
	exports.foo = 123;
});

//#endregion
//#region entry.js
var import_foo = __toESM(require_foo());
console.log(`bar: `, import_foo.bar); // you could see here, it still use  `import.bar`

// snip
```

```